web/satellite/src/components/objects/EncryptData.vue

```diff
@@ -85,11 +87,14 @@ export default class EncryptData extends Vue {
     public isLoading = false;
     public passphrase = '';
 
+    public readonly analytics: AnalyticsHttpApi = new AnalyticsHttpApi();
+
     /**
      * Sets passphrase from child component.
      */
     public navigateToCLIFlow(): void {
         this.$store.commit(APP_STATE_MUTATIONS.SET_ONB_AG_NAME_STEP_BACK_ROUTE, this.$route.path);
+        this.analytics.pageVisit(RouteConfig.OnboardingTour.with(RouteConfig.OnbCLIStep.with(RouteConfig.AGName)).path);
```

Comment:
where is `RouteConfig.OnboardingTour.with(RouteConfig.OnbCLIStep.with(` coming from?